pkg/rootless/rootless_linux.go

```diff
@@ -394,7 +396,9 @@ func becomeRootInUserNS(pausePid, fileToRead string, fileOutput *os.File) (_ boo
 			}
 
 			if err := unix.Kill(int(pidC), s.(unix.Signal)); err != nil {
-				logrus.Errorf("failed to kill %d", int(pidC))
+				if err != unix.ESRCH {
+					logrus.Errorf("failed to kill %d: %v", int(pidC), err)
```

Comment:
One habit that has served me well over the years is: always write unique error messages, so it's possibly to identify the exact place in the code where a failure happened. Would it make sense here to disambiguate these two, by adding a short relevant context descriptor to each error string?

Reply:
good idea, made them different